StripePaymentProcessor.cs

```diff
@@ -88,7 +93,7 @@ public ProcessPaymentResult ProcessPayment(ProcessPaymentRequest processPaymentR
             StripeChargeCreateOptions chargeOptions = new StripeChargeCreateOptions()
             {
                 Amount = paymentAmount,
-                Currency = "gbp",
+                Currency = _currencyService.GetCurrencyById(_currencySettings.PrimaryStoreCurrencyId)?.CurrencyCode,
```

Comment:
Copied from [Payments.PayPalStandard](https://github.com/nopSolutions/nopCommerce/blob/072ec444a4d38ea655c67b44d0ddcaf40d819315/src/Plugins/Nop.Plugin.Payments.PayPalStandard/PayPalStandardPaymentProcessor.cs#L209), though this is more robustly handled by [Payments.Square](https://github.com/nopSolutions/nopCommerce/blob/072ec444a4d38ea655c67b44d0ddcaf40d819315/src/Plugins/Nop.Plugin.Payments.Square/SquarePaymentMethod.cs#L173-L180).